python/pyarrow/array.pxi

```diff
@@ -1902,6 +1902,7 @@ cdef class ListArray(BaseListArray):
             If not specified, a default ListType with the values' type is
             used.
         pool : MemoryPool
+        mask : Array (bool type)
```

Comment:
I would spell it "boolean" (to avoid confusions with Python's `bool` type) and would also describe the argument a bit more:
```suggestion
        mask : Array (boolean type), optional
            ... (fill this :-))
```